org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/wizards/NewTypeWizardPage.java

```diff
@@ -3529,6 +3789,7 @@ public IStatus getSuperInterfaceStatus() {
 	}
 
 	/**
+	 * Getter.
```

Comment:
Please avoid such Boiler plate comments and refactorings. they make this commit unnecessary complicated to review.